scripts/upload_version.sh

```diff
@@ -24,4 +24,5 @@ DIRECTORY=$2
 BUCKET=$3
 
 export GOPATH=$(go env GOPATH)
-${GOPATH}/bin/updater send -s "${DIRECTORY}" -c "${CHANNEL}" -b "${BUCKET}"
+export PATH=${PATH}:${GOPATH}/bin
+updater send -s "${DIRECTORY}" -c "${CHANNEL}" -b "${BUCKET}"
```

Comment:
Could you please explain why the changes to this file are needed ?
I can see why this would work, but unless you need the PATH being set from within the updater, I don't see any benefit here.

Reply:
We made this change because in the pipeline we're storing built binaries, and then running the uploading on another host later in the process. We have our makefile set up to download those binaries and add them to the path. We left `export PATH=${PATH}:${GOPATH}/bin` in here as a failsafe in case someone calls this script not from the make targets so the go bin will still be part of the path.

Reply:
I see. so you basically saying that you want to give priority to the updater in the path before taking the updater from the GOPATH. In this case, I have only one request - make it use the GOPATH1 ( i.e. the first path in the GOPATH)